src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_appengine.v1beta5.json.baseline

```diff
@@ -0,0 +1,352 @@
+
+# PRE-REQUISITES:
+# ---------------
+# 1. If not already done, enable the Google App Engine Admin API and check the quota for your project at
+#    https://console.developers.google.com/apis/api/appengine_component/quotas
+# 2. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from
+#    https://cloud.google.com/sdk/ and run 'gcloud beta auth application-default login'
+# 3. To install the client library and Application Default Credentials library, run:
+#    'gem install google-api-client'
+#    'gem install googleauth'
+require 'googleauth'
+require 'google/apis/appengine_v1beta5'
+
+service = Google::Apis::AppengineV1beta5::AppengineService.new
+service.authorization = Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
+
+# TODO: Change placeholders below to values for parameters to the 'get_app' method:
```

Comment:
Use the language in the Go comment (corrected) for consistency:

TODO: Change placeholders below to appropriate parameter values for the 'delete' method: